frontend/public/style/_common.scss

```diff
@@ -121,7 +121,7 @@
 }
 
 .cos-error-title {
-  color: $red;
+  color: $color-pf-red;
```

Comment:
Duplicate rule also in `_status-box.scss`

Reply:
Both are needed.  This one is the generic one.  The `_status-box.scss` one is needed to override the color change 6 lines up.